grades/tests.py

```diff
@@ -1,3 +1,50 @@
-# from django.test import TestCase
+import pytest
+from grades.models import Grade, Course
+from django.contrib.auth.models import User
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestGrade:
+    def test_addGradeUserCourse(cls):
+        out = Grade.objects.filter(semester='A')
+        assert list(out.values_list('user', 'course', 'grade', 'semester', 'year')) == [
+            (1, 1, 85, 'A', 2021),
+        ]
+
+    def test_removeGrade(cls):
+        user3 = User.objects.create_user(username="idan", password="123456")
+        c = Course(course_name="NLP", credinitials="3")
+        c.save()
+        g = Grade(user=user3, course=c, grade=90, semester='A', year=2021)
+        g.save()
```

Comment:
You have a classmethod called addGradeUserCourse which does the same thing as this. Why not use that?